recipes/spot/build.sh

```diff
@@ -0,0 +1,5 @@
+./configure --prefix=$PREFIX --disable-devel --enable-optimizations
+make
+make install-strip
```

Comment:
Is there a `make check/test` we can run here?

Reply:
Yes, I can add `make check` there.

Reply:
Now the builds take 5 hours instead of 2.  (Our test suite is quite large.)

Reply:
Maybe we can run a subset of it? We do have 6 hours on Azure but I do not want to abuse it :smile: 

Anyway, let's leave that to the feedstock. Merging this!